.kitchen.yml

```diff
@@ -20,15 +20,9 @@ platforms:
   - name: ubuntu-14.04
     run_list:
     - recipe[apt]
-    attributes:
-      postgres:
-        version: "9.3"
-  - name: centos-6.5
```

Comment:
I think, there is no need to change the name `centos-6.5` -> `opscode-centos-6.5`, isn't it?